README.md

```diff
@@ -1219,6 +1219,14 @@ If you are using Linux on the desktop, you may want to use `/usr/bin/pinentry-gn
     $ pkill ssh-agent ; pkill gpg-agent ; \
       eval $(gpg-agent --daemon --enable-ssh-support --use-standard-socket \
       --log-file ~/.gnupg/gpg-agent.log --write-env-file)
+      
+Zsh note - if you're using zsh, add gpg-agent to your .zshrc plugins:
```

Comment:
I don't work on this document, I watch the repo out of curiosity :-)

I would mention this is oh-my-zsh specific as there are people who use zsh on it's own (Me 👋 ).